### PR TITLE
Fix #145044 mechacomic.jp

### LINKED ISSUE #145044: mechacomic.jp
### Issue URL (Social Widget)
 [https://mechacomic.jp/books/106454/volume/18](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmechacomic.jp%2Fbooks%2F106454%2Fvolume%2F18)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.5
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #145044 

### Add your comment and screenshots

 

<details><summary>screenshot - PC</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -2636,7 +2636,7 @@ svtv.org##.quiz-test__socials-header
 manga.line.me##.mdGHD01Share
 manga.line.me##.mdMNG01Share
 sukima.me##.share-description
-comics.mecha.cc##.area_btn_sns
+mechacomic.jp,comics.mecha.cc##.area_btn_sns
```
Comment (piquark6046) [inline SocialFilter/sections/specific.txt:2639]: Please hide the related text too.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -155,7 +155,7 @@ washingtonpost.com#?#div[data-qa="article-actions"] > div[class^="wpds-c-"] > di
 adme.media#?#article > div[class] > div[class]:has(> div[class]:contains(Поделиться этой статьёй))
 forbes.kz#?#.watches > span:contains(Поделиться этой публикацией)
 quantamagazine.org#?#.post__sidebar__content > div.post__sidebar__content__inner:has(> h3:contains(Share this article))
-mechacomic.jp#?##main > section.c-content > div.area_btn_sns:upward(1)
+mechacomic.jp#?##main section.c-content > div.area_btn_sns:upward(1)
```
Comment (Alex-302) [inline SocialFilter/sections/css_extended.txt:158]: Is `main` required?

Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:158]: `main` was included in the original rule.
so I followed.

**Comment by SKEIDs** [comment]
already fixed by https://github.com/AdguardTeam/AdguardFilters/commit/db82f894b9872f834d1cc61883fada5c86ece263

**Pull request closed without merging**